packages/web-frontend/cypress.json

```diff
@@ -1,9 +1,10 @@
 {
   "baseUrl": "http://localhost:8088",
-  "integrationFolder": "cypress/e2e",
+  "experimentalFetchPolyfill": true,
   "ignoreTestFiles": "snapshots.js",
-  "viewportHeight": 900,
-  "viewportWidth": 1400,
+  "integrationFolder": "cypress/e2e",
+  "requestTimeout": 20000,
```

Comment:
This is the only added field here (the rest are just sorted).

The original timeout is `5000`, which is certainly not enough for some reports. We may find out that we need to adjust this even more, or leave it there as a canary for reports taking too long to load

Related to this, a positive side effect of having e2e tests is that they can serve as a rough benchmarks:

![image](https://user-images.githubusercontent.com/20692464/97365287-8e677400-18f9-11eb-9742-cff05e792a98.png)